csunplugged/topics/management/commands/_CurriculumAreasLoader.py

```diff
@@ -56,10 +56,19 @@ def load(self):
                     "Curriculum Area"
                 )
 
+            curriculum_area_colour = curriculum_area_data.get("colour", None)
+            if curriculum_area_colour is None:
+                raise MissingRequiredFieldError(
+                    self.structure_file_path,
+                    ["colour"],
```

Comment:
Why don't we switch it to a specific field? Wouldn't that be easier for authors?